src/app_service/service/activity_center/dto/notification.nim

```diff
@@ -13,10 +13,22 @@ type ActivityCenterNotificationType* {.pure.}= enum
   Mention = 3,
   Reply = 4,
   ContactRequest = 5
+  CommunityInvitation = 6
+  CommunityRequest = 7
+  CommunityMembershipRequest = 8
```

Comment:
What's the difference between these two?

Reply:
One is for sender side, another for receiver. I also don't like those naming, but i took them from figma: https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1267%3A320353